src/client/.babelrc

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
Note: this file is now no longer needed, as the client-side babel config lives inside `webpack.common.js`.